<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Metadata Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="objectName">
       <string>titleLabel</string>
      </property>
      <property name="text">
       <string>üì∑ –ê–Ω–∞–ª–∏–∑–∞—Ç–æ—Ä –º–µ—Ç–∞–¥–∞–Ω–Ω—ã—Ö –∏–∑–æ–±—Ä–∞–∂–µ–Ω–∏–π</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controlLayout">
      <item>
       <widget class="QLabel" name="folderLabel">
        <property name="text">
         <string>–ü–∞–ø–∫–∞:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="folderLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>–í—ã–±–µ—Ä–∏—Ç–µ –ø–∞–ø–∫—É —Å –∏–∑–æ–±—Ä–∞–∂–µ–Ω–∏—è–º–∏...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>–û–±–∑–æ—Ä...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="analyzeButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>–ê–Ω–∞–ª–∏–∑–∏—Ä–æ–≤–∞—Ç—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>–°—Ç–æ–ø</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>–ü–æ–∏—Å–∫:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchText">
        <property name="placeholderText">
         <string>–ü–æ–∏—Å–∫ –ø–æ –∏–º–µ–Ω–∏ —Ñ–∞–π–ª–∞ –∏–ª–∏ —Ñ–æ—Ä–º–∞—Ç—É...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="objectName">
       <string>statusLabel</string>
      </property>
      <property name="text">
       <string>–ì–æ—Ç–æ–≤ –∫ —Ä–∞–±–æ—Ç–µ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="overviewTab">
       <attribute name="title">
        <string>üìä –û–±–∑–æ—Ä –∏–∑–æ–±—Ä–∞–∂–µ–Ω–∏–π</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
        <item>
         <widget class="QLabel" name="detailsLabel">
          <property name="text">
           <string>–î–µ—Ç–∞–ª—å–Ω–∞—è –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—è (–¥–≤–æ–π–Ω–æ–π –∫–ª–∏–∫ –ø–æ —Å—Ç—Ä–æ–∫–µ):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="detailsTextEdit">
          <property name="maximumHeight">
           <number>250</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statisticsTab">
       <attribute name="title">
        <string>üìà –°—Ç–∞—Ç–∏—Å—Ç–∏–∫–∞</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="statsLabel">
          <property name="text">
           <string>–°—Ç–∞—Ç–∏—Å—Ç–∏–∫–∞ –±—É–¥–µ—Ç –æ—Ç–æ–±—Ä–∞–∂–µ–Ω–∞ –ø–æ—Å–ª–µ –∞–Ω–∞–ª–∏–∑–∞</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
